src/comp_loinc/schema/code_schema.yaml

```diff
@@ -198,7 +198,7 @@ slots:
     slot_uri: loinc:loinc_number
     annotations:
       owl: AnnotationAssertion
-  status:
+  status:  # todo: define valid values?: ACTIVE, DEPRECATED, DISCOURAGED, TRIAL
```

Comment:
Minor todo: Define the `range` on some of these fields like `status` and `class_type`?